### DISCUSSION TITLE
Update sourceUrl

### DISCUSSION THREAD

**Discussion by RahulGautamSingh** [spockframework/spock#1570] [General]
Related Discussion: https://github.com/renovatebot/renovate/discussions/20276
Can the sourceUrl be updated as the current one leads nowhere.

**Comment by szpak** [comment]
I had a similar problem recently with Renovate :). In fact "github.spockframework.org" even is not resolveable at the moment:
```
$ ping github.spockframework.org
ping: github.spockframework.org: Name or service not known
```

Could you provide a PR for it in:
https://github.com/spockframework/spock/blob/0c5531aa8f9b357eee226c06903416e4be108ff8/gradle/publishMaven.gradle#L33
?

**Reply by RahulGautamSingh** [reply]
Sure :)
Btw I came from Renovate after you opened the discussion 😅

**Reply by szpak** [reply]
> Sure :) Btw I came from Renovate after you opened the discussion sweat_smile

Yup, I've got a notification. I just wanted to say here, that you are not the only one with that issue :).
